dotnet/targets/Xamarin.Shared.Sdk.props

```
@@ -50,6 +50,7 @@
 			* https://github.com/dotnet/sdk/blob/c5a58bc6c3eb2b236b314e6d17a89a537459890c/src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.FrameworkReferenceResolution.targets#L102
 		-->
 		<_IsTrimmingEnabled>true</_IsTrimmingEnabled>
+		<GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
```

This should be contingent upon the project being an extension project (otherwise you end up enabling it for library projects as well, which you don't want):

```suggestion
		<GenerateRuntimeConfigurationFiles Condition="'$(IsAppExtension)' == 'true'">true</GenerateRuntimeConfigurationFiles>
```

Also please add the explanation from your PR description as a comment to the file here, so that the next person who sees this doesn't have to look through git to understand what's happening.